### DISCUSSION TITLE
Promoted attributes formatting

### DISCUSSION THREAD

**Discussion by darkpoetry** [TriliumNext/Trilium#3800] [General]
Hello,

Is it possible to set the field size or other formatting information for promoted attributes? I currently have some labels that contain only one letter for example but are about 30 characters long when displayed.

Thanks!

**Comment by zadam** [comment]
Hi, there isn't any such formatting option. You could do some hacking with custom CSS if it would be worth the trouble ... The elements have `data-attribute-name` HTML attributes on which the styles could be used.
